### PR TITLE
replace pipelines api call with dashboard's redux selectors & actions

### PR DESCRIPTION
**Pull request by carlos-logro**
issue: #137 

# Changes
removed & replaced the api calls for getting namespaces, service accounts and pipelines with the redux selectors & actions that were passed as props to the extension component. Also updated all necessary tests. 

Its worth mentioning that the api call to get webhook secrets was left because at the moment the getSecrets action did not return the secrets created for webhooks.

### CODE REVIEW

**Comment by carlos-logro** [comment]
@AlanGreene this PR only includes everything regarding the pipeline api call. The third one & last one will be for the service account one.

File: webhooks-extension/src/components/WebhookCreate/WebhookCreate.js [hunk 1]
```diff
@@ -86,10 +85,8 @@ class WebhookCreatePage extends Component {
   }
 
   async fetchPipelines(namespace) {
-    let pl;
     try {
-      pl = await getPipelines(namespace);
-      this.setState({apiPipelines: pl})
+      this.props.fetchPipelines(namespace);
     } catch (error) {
```
Comment (AlanGreene) [inline webhooks-extension/src/components/WebhookCreate/WebhookCreate.js]: the error should already be caught by the action and is exposed via the `getPipelinesErrorMessage` selector, no need for the try-catch here

Comment (carlos-logro) [reply webhooks-extension/src/components/WebhookCreate/WebhookCreate.js]: done. Added logic to output notification if error happens and a test case.

**Pull request merged**